### ISSUE TITLE
docs: "Best Practices > Accessibility" missing mention of Angular Aria

### ISSUE THREAD

**Issue by michael-small** [angular/angular#68919]
### Describe the problem that you experienced

The "Best Practices > Accessibility" page is missing a mention of Angular Aria. 

### Enter the URL of the topic with the problem

https://next.angular.dev/best-practices/a11y

### Describe what you were looking for in the documentation

I would assume with the package going stable that there would be a mention in that page.

### Describe what you want to experience that would fix the problem

Some reference to the Angular Aria package and its documentation is warranted on this greater Accessibility page. That said, there is a lot of ways this could be handled on that page, otherwise I would make a PR.

### If this problem caused an exception or error, please paste it here

```true

```

### If the problem is browser-specific, please specify the device, OS, browser, and version

```true

```

### Provide any additional information here in as much as detail as you can

```true

```
